lang/cs/Org.Apache.REEF.Common/Org.Apache.REEF.Common.csproj

```diff
@@ -39,14 +39,14 @@ under the License.
     <Reference Include="protobuf-net">
       <HintPath>$(PackagesDir)\protobuf-net.$(ProtobufVersion)\lib\net40\protobuf-net.dll</HintPath>
     </Reference>
+    <Reference Include="System" />
```

Comment:
This is a relic of the Nuget Package Manager -- these were actually there before; when the package manager rewrote the .csproj file, it moved these lines around, so this actually isn't a change.

That said I'm not entirely sure we need System.Core in this project. However, I'd rather not change that in this PR; I'd like to restrict this PR to just the Reactive library.